cadc-log/src/main/java/ca/nrc/cadc/log/WebServiceLogInfo.java

```diff
@@ -165,6 +166,29 @@ private String getPreamble() {
         return sb.toString();
     }
 
+    // handle Lists, refer to GroupLogInfo.addedMembers
+    // currently only List<String> is handled
+    private void populateList(StringBuilder sb, Object o) {
```

Comment:
Why not send in the List<?> rather than the object?  Then you won't need to re-cast.

Reply:
Changed.